gitops/commit.sh

```diff
@@ -45,7 +45,7 @@ else
     MDS_GIT_HASH=$(cat $TARGET_APP/overlays/$SOURCE_ENV/*deployment.patch.yaml | grep git-commit -m1 | cut -f2 -d ":" | xargs | cut -d "-" -f 3)
     echo
     # Replace the commit sha in env vars in overlay patch
-    sed -i "s^git-commit.*^git-commit-$MDS_GIT_HASH^" $TARGET_APP/overlays/$TARGET_ENV/deployment.patch.yaml
+    sed -i "s^git-commit.*^git-commit-$MDS_GIT_HASH^" $TARGET_APP/overlays/$TARGET_ENV/*deployment.patch.yaml
```

Comment:
Do we need the * wildcard. Does it work as intended?

Reply:
We will if we want to include different patches for multi resource configurations (e.g. docman, postgres).